### PR TITLE
added support for temperature/humidity/pressure and leak detection sensor

### PR DESCRIPTION
**Pull request by Raidok**
Added support to these 2 new devices:

Updated temperature & humidity sensor which now measures air pressure too:

Water leak detection sensor:

Cheers!

### CODE REVIEW

File: example.js [hunk 1]
```diff
@@ -55,9 +55,18 @@ aqara.on('gateway', (gateway) => {
...
       case 'sensor':
-        console.log(`  Sensor (temperature:${device.getTemperature()}C rh:${device.getHumidity()}%)`)
+        console.log(`  Sensor (temperature:${device.getTemperature()}C rh:${device.getHumidity()}%${device.getPressure() != null ? ' pressure:' + device.getPressure() + 'kPa': ''})`)
         device.on('update', () => {
-          console.log(`${device.getSid()} temperature: ${device.getTemperature()}C rh:${device.getHumidity()}%`)
+          console.log(`${device.getSid()} temperature: ${device.getTemperature()}C rh:${device.getHumidity()}%${device.getPressure() != null ? ' pressure:' + device.getPressure() + 'kPa' : ''}`)
+        })
+        break
+      case 'leak':
+        console.log(`  Leak sensor`)
```
Comment (marvinroger) [inline example.js]: Can you do like the other sensors? The device should have a `isLeaking()` method and emit events *on update*, not initially.

**Comment by marvinroger** [comment]
I just started a review; other than that, thanks a lot, that's great! 😇

File: example.js (hunk 1 above)
Comment (Raidok) [reply example.js]: For some reason I thought the leak sensor is more similar to the switch but now your idea sounds more reasonable :)
Made the changes

**Comment by marvinroger** [comment]
Thanks!

**Pull request merged**